src/checks/lowercase_key.rs

```diff
@@ -19,14 +21,22 @@ impl Check for LowercaseKeyChecker {
         if key.to_uppercase() == key {
             None
         } else {
+            let warning_text = self.template.replace("{}", &key);
+
             Some(Warning::new(
                 line.clone(),
-                self.template.replace("{}", &key),
+                self.message(&warning_text),
```

Comment:
Please, run the `cargo fmt` command.
https://github.com/mgrachev/dotenv-linter/pull/131/checks?check_run_id=560584416